Source/Extensions/Blazorise.Markdown/Markdown.razor.cs

```diff
@@ -200,6 +200,9 @@ protected async Task<T> ExecuteAfterRenderAsync<T>( Func<Task<T>> action, Cancel
         /// <returns>Markdown value.</returns>
         public async Task<string> GetValueAsync()
         {
+            if ( Rendered )
+                return await JSModule.GetValue( ElementId );
+
             return await ExecuteAfterRenderAsync( async () => await JSModule.GetValue( ElementId ) );
```

Comment:
These `ExecuteAfterRenderASync` lines make sense for the getters also?

Choose one or another (don't think it matters which here),
`await InvokeAsync` vs `await ExecuteAfterRenderAsync`

Reply:
We have no idea what users can think of so it's better to have them.